qemu/tests/rng_bat.py

```diff
@@ -61,6 +65,10 @@ def is_dev_used_by_qemu(dev_file, vm_pid):
         session = utils_test.qemu.windrv_check_running_verifier(session, vm,
                                                                 test, driver_name,
                                                                 timeout)
+        if not system.file_exists(session,
+                                  "c:\\random_%PROCESSOR_ARCHITECTURE%.exe"):
```

Comment:
It is used to check the target file,   can use:
`if not system.file_exists(session, read_rng_cmd)`

Reply:
read_rng_cmd sometimes is a loop of random.exe, so read_rng_cmd can not be used

Reply:
@lijinlijin i did not get the point about `loop`, from the configuration, `read_rng_cmd  = "c:\random_%PROCESSOR_ARCHITECTURE%.exe`. How can it be a loop ?
Could you explain more, thanks!

Reply:
for example, in viorng_in_use.cfg: 
read_rng_cmd = for /l %i in (1, 1, 1000) do "c:\random_%PROCESSOR_ARCHITECTURE%.exe"

Reply:
@lijinlijin Got it, thanks! 
IMO, it is better to use a parameter instead of a str that should keep same with a parameter.  Other wise, we need update both configuration and script if it changed.  So i suggest moving the `for` into script.

Reply:
This makes sense.
Will modify it, thanks